Detour/Include/DetourNavMeshQuery.h

```diff
@@ -194,6 +194,24 @@ class dtNavMeshQuery
 					  const dtQueryFilter* filter,
 					  dtPolyRef* path, int* pathCount, const int maxPath) const;
 
+	/// Finds a path from the start polygon to the closest (by geodesic distance) to any of the goals.
+	/// Only traverse bidirectional edges.  Therefore it is not gaurenteed to find the shortest path 
+	/// or closest goal in meshes with directed offmesh connections.
+	///  @param[in] 	numGoals    The number of goals
+	///  @param[in]		startRef	The refrence id of the start polygon.
+	///  @param[in]		endRefs		List of the reference ids of the end polygons. [id * @p numGoals]
+	///  @param[in]		startPos	A position within the start polygon. [(x, y, z)]
+	///  @param[in]		endPoses	List of end positions within position within the end polygons. [(x, y, z) * @p numGoals]
+	///  @param[in]		filter		The polygon filter to apply to the query.
+	///  @param[out]	path		An ordered list of polygon references representing the path. (Start to end.) 
+	///  							[(polyRef) * @p pathCount]
+	///  @param[out]	pathCount	The number of polygons returned in the @p path array.
+	///  @param[in]		maxPath		The maximum number of polygons the @p path array can hold. [Limit: >= 1]
+	dtStatus findBidirPathToAny(const int numGoals, dtPolyRef startRef, const dtPolyRef* endRefs,
+					  const float* startPos, const float* endPoses,
+					  const dtQueryFilter* filter,
+					  dtPolyRef* path, int* pathCount, const int maxPath) const;
```

Comment:
I think I would prefer the parameter order to be:
`startRef, startPos, endRefs, endPoses, numGoals, filter, path, pathCount, maxPath`
Even though it is inconsistent with `findPath` it is more consistent with order APIs.
However I am open to comments on this. Not sure what would be the easiest for users.

Reply:
Makes sense to do it the way you suggested if that is how lists are commonly specified in the rest of the API.

Reply:
I think you missed the rest of the reordering (bunching up start/end).

Reply:
Yep!  I most certainly did.  Fixed.